extensions/amp-story/amp-story.md

```diff
@@ -362,6 +374,48 @@ Usage: `<amp-story ... supports-landscape>...</amp-story>`
   </amp-anim>
 </figure>
 
+### Live story
+
+Using the `live-story` attribute, you can append new pages to a story for users to see in real-time. If the user is on the last page it will show a notification saying there are new pages and the progress bar will be updated.
```

Comment:
Avoid passive voice. Update to:

Use the `live-story` attribute to append new pages to a story for users to see in real-time. 

This attribute will show a notification of new pages and update the progress bar to users on the last page.